### PR TITLE
Feature Matrix for the Java SK (Experimental)

### PR DESCRIPTION
**Pull request by brunoborges**

### CODE REVIEW

File: FEATURE_MATRIX.md [hunk 1]
```diff
@@ -1,64 +1,76 @@
...
+| Postgres (Memory)                 | ✅ | ❌ | ❌ | Vector optimized (required the [pgvector](https://github.com/pgvector/pgvector) extension) |
+| Azure Cognitive Search            | ❌ | ❌ | ❌ | |
+| MsGraph                           | ✅ | ❌ | ❌ | Contains connectors for OneDrive, Outlook, ToDos, and Organization Hierarchies |
+| Document Skills                   | ✅ | ❌ | ❌ | Currently only supports Word documents |
+| OpenAPI                           | ✅ | ❌ | ❌ | |
+| Web Skills                        | ✅ | ❌ | ❌ | |
+
+## Design Choices
 
 The overall architecture of the core kernel is consistent across Python and C#,
```
Comment (markwallace-microsoft) [inline FEATURE_MATRIX.md:75]: @brunoborges can we include Java in this sentence? Or if not explain why.

**Review by markwallace-microsoft — APPROVED** [review]
Just one comment but otherwise LGTM

**Review by johnoliver — APPROVED** [review]

File: FEATURE_MATRIX.md (hunk 1 above)
Comment (brunoborges) [reply FEATURE_MATRIX.md]: Fixed in [8e560df](https://github.com/microsoft/semantic-kernel/pull/1189/commits/8e560dffb56e668b8cfee26d049fe8ff71347230)

**Comment by lemillermicrosoft** [comment]
Shouldn't this target the `experimental-java` branch until it lands in `main`?

**Comment by markwallace-microsoft** [comment]
> Shouldn't this target the `experimental-java` branch until it lands in `main`?

@lemillermicrosoft two things:

1. We want this in main to raise awareness that we have an experimental Java implementation and the progress we are making
2. Changes to experimental-java are all in the java folder. Changes outside of the Java folder should go to main so we avoid merge conflicts when we automate updating the experimental-java branch.

**Review by dluc — APPROVED** [review]

**Review by markwallace-microsoft — APPROVED** [review]
@brunoborges see my comment for a suggestion on a follow up PR

**Review by johnoliver — APPROVED** [review]

**Pull request merged**